autodocs.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+LINT=
+GENERATE=
+
+while getopts :gl opt
+do
+    case "${opt}" in
+        l)
+            LINT=1
+            ;;
+        g)
+            GENERATE=1
+            ;;
+        \?)
+            echo "ERROR: unknown parameter \"$OPTARG\""
+            exit 1
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+IFS=$'\n'
+# create an array of all unique directories containing .tf files
+mapfile -t directories < <(find . -name '*.tf' -exec dirname {} \; | sort -u)
```

Comment:
Should be able to prefix this line with the IFS, to avoid changing the IFS for other commands...

```suggestion
IFS=$'\n' mapfile -t directories < <(find . -name '*.tf' -exec dirname {} \; | sort -u)
```

Reply:
copypasta error